apps/server/lib/google-drive/src/internal/detect-platform.ts

```
@@ -0,0 +1,197 @@
+	if (arch) return `other:${arch}`;
+	return "unknown";
+};
+
+const normalizePlatform = (platform: string): PlatformName => {
+	// Node platforms:
+	// - https://nodejs.org/api/process.html#processplatform
+	// Deno platforms:
+	// - https://doc.deno.land/deno/stable/~/Deno.build
+	// - https://github.com/denoland/deno/issues/14799
+
+	platform = platform.toLowerCase();
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)